config/unicorn.conf.rb

```diff
@@ -94,77 +94,131 @@
         STDERR.puts "#{Time.now}: Issuing stop to sidekiq"
         Demon::Sidekiq.stop
       end
+    end
 
-      class ::Unicorn::HttpServer
-        alias :master_sleep_orig :master_sleep
+    puts "Starting up email sync"
```

Comment:
Just checking... is it safe to run multiple `Demon::EmailSync` at the same time? It looks like there is one Demon per host, which means we will have multiple instances running against the same database.